### ISSUE TITLE
dts hd ma 2.0 chanel count are wrong

### ISSUE THREAD

**Issue by letianxia** [mbunkus/mkvtoolnix#2035]
When I converted a mkv-formated video with dts hd ma 2.0 to another, the result chanel count were not 2 but 6, so, it's very annoying.

**Comment by mkver** [comment]
You should upload the original mkv-formated video to [the ftp-server](https://github.com/mbunkus/mkvtoolnix/wiki/FTP-server).

**Comment by mbunkus** [comment]
As pointed out, I'll need a sample file for investigation; otherwise I'll close this issue again.

**Comment by letianxia** [comment]
You can use any video with dts hd ma 2.0 or doby 2.0 audio track to convert to a mkv-formated video. And then, you use potplayer to play it, and press "Tab" keyboard key to display the video's audio format, thus you can see the result is "chanel count were not 2 but 6". Here is sample original video: "Mannequin On the Move (1991) 1080p BluRay AVC DTS HD MA 2 0 FGT", and it's download webpage is "https://torrentproject.se/?t=Mannequin+1991"

**Comment by letianxia** [comment]
If you insist to ask me for uploaded sample files, I will tell you that I have few problems: (1) these days I'm really very busy, (2) size of these files is so big, (3) my upload speed is not ideal. And I will upload it someday, but not so sure.

**Comment by letianxia** [comment]
Audio channel count of the two movies: "Mannequin 1987" and "Mannequin 1991" are all 2, you can download any type or any size of these movies with dts or doby audio track as the original sample movie.

**Comment by mbunkus** [comment]
Thanks for the info, but I'm not particularly keen on downloading Blu-ray rips illegally. Therefore I'm still asking you to upload a small sample. This doesn't have to be a lot: the first 50 MB or so of the main M2TS file should suffice for my analysis. If you're using WinSCP for the upload, then just start the upload and abort it after ~50 MB have been uploaded. The partial file will stay on the server, and that will suffice for me.

Thanks.

**Comment by letianxia** [comment]
My original file has been uploaded, it's name is "Nothing's Gonna Stop Us Now.HDMA2 0.48x24.Starship Mannequin 1991.dts".  You can use your "MKVToolNix GUI" to convert it to a mka file, then use potplayer to play it and see result channel count of it.

**Comment by mbunkus** [comment]
Thanks. I'll take a look at it.

**Comment by mbunkus** [comment]
I'm sorry, but I cannot find anything wrong with mkvmerge with this file. After multiplexing it with mkvmerge v13.0.0 most software identifies the audio as stereo (= 2 channels) just fine. These include:

* VLC ("Tools" menu → "Codec information")
* MPC-HC (context menu → "Properties")
* ffprobe (from the ffmpeg package)
* MediaInfo

The only odd man out here is Potplayer 1.7.2710 which states "Channels: 6 → 2". That's probably what you mean. However, saying that this is a problem is most likely either a misinterpretation of the information shown, or it's a but in Potplayer.

In short; there's no bug in mkvmerge here. The channel count is correct.
